packages/core/src/configuration/ConfigurationHelper.ts

```diff
@@ -17,10 +17,18 @@ export class ConfigurationHelper {
 
   /**
    * Get a service configuration object, provided an existing id is given
-   * @param type string - The type of the service
+   * @param type - The type of the service
    */
   public getConfig(type: string): ServiceConfiguration | undefined {
```

Comment:
should this method actually return an array?

Reply:
Very good point!

Reply:
This method filters the `services` array and returns only one ServiceConfig, ready to be used. Is there any reason to return an array instead?

Reply:
yes, there could be multiple `custom-runtime-connector` services and they all will have the same types, but each with different id.

Reply:
I don't understand why we get the first one then (`[0]`) if there could be many of them, I think we should still be filtering by `id`.

Reply:
@josemigallas see conversation on the list for reasoning. Happy to explain that as well.